tests/foreman/maintain/test_health.py

```diff
@@ -610,8 +615,6 @@ def test_positive_health_check_foreman_proxy_verify_dhcp_config_syntax(sat_maint
                       is set other than `dhcp_isc`, and also not on DHCP disabled Satellite.
 
     :CaseImportance: Medium
-
-    :CaseAutomation: Automated
```

Comment:
Not sure what this means ;)

Reply:
This indicates that the test is automated, but since CaseAutomation is already defined at the top of the module, it’s redundant here